tiled_test.go

```diff
@@ -218,3 +220,122 @@ func TestLoader(t *testing.T) {
 
 	assert.Equal(t, []string{mapFile, filepath.Join(GetAssetsDirectory(), "..", "README.md")}, fs.AttemptedOpen)
 }
+
+func TestParseHexColor(t *testing.T) {
```

Comment:
Please add also test case for upper case hex color

Reply:
done